easybuild/easyconfigs/g/GTK4/GTK4-4.6.3-GCC-11.3.0.eb

```diff
@@ -0,0 +1,82 @@
+easyblock = 'Bundle'
+
+name = 'GTK4'
+version = '4.6.3'
```

Comment:
https://gitlab.gnome.org/GNOME/gtk/-/tags/4.7.0 has been released since this PR was created.